plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/delete/TestDeletionVectors.java

```diff
@@ -47,13 +47,29 @@ public void testUuidStorageType()
         assertThat(bitmaps.contains(2)).isFalse();
     }
 
+    /**
+     * @see deltalake.p_type_deletion_vectors
+     */
     @Test
-    public void testUnsupportedPathStorageType()
+    public void testPathStorageType()
```

Comment:
I'm sorry to say this, but this test isn't helpful. Please reuse the existing bin file and add a product test instead. We want to verify the E2E (=query based) result.

Reply:
Sure! let me update the test, which class is suitable for adding the test

Reply:
We can use `TestDeltaLakeDeleteCompatibility` or `TestDeltaLakeCloneTableCompatibility`.